handler/push.go

```diff
@@ -36,6 +36,11 @@ import (
 	"github.com/prometheus/pushgateway/storage"
 )
 
+const (
+	pushMetricName = "push_time_seconds"
+	pushMetricHelp = "Last unixtime when this group was changed in the pushgateway."
```

Comment:
In the spirit of the above, I suggest "Last Unix time when this group was changed in the Pushgateway."